spec/features/request_a_mentor_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+feature 'User can request a mentor' do
+
+  let(:mentee) { create :user, :not_on_waitlist }
```

Comment:
see mentee factory comment in mentorship factory `let(:mentee) { create :mentee }`